extras/cloudshell/tutorial.md

```diff
@@ -9,44 +9,49 @@ You'll be able to run Bank of Anthos on:
 
 Let's get started!
 
-## Kubernetes cluster setup
+## Select a GCP project
 
-Set up a Kubernetes cluster using the instructions below for either **minikube** or **GKE**.
+In order to create a Google Kubernetes Engine cluster, you'll need to create a Google Cloud Platform project or use an existing project.
 
----
-### Minikube
+Note: If you choose to deploy to minikube instead of GKE, you can skip this step.
 
-Minikube creates a local Kubernetes cluster on Cloud Shell.
+<walkthrough-project-setup></walkthrough-project-setup>
 
-1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">minikube</walkthrough-editor-spotlight> on the status bar located at the bottom of the editor window.
+## Kubernetes cluster setup
 
-2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**. 
+This section shows you how to set up a Kubernetes cluster using either **minikube** or **GKE**.
 
-3. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+1. Navigate to the <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer">**Clusters**</walkthrough-editor-spotlight> explorer in the <walkthrough-editor-spotlight spotlightId="activity-bar-cloud-k8s">Cloud Code - Kubernetes sidebar</walkthrough-editor-spotlight> using the left side Activity bar.
 
-Once minikube has finished starting, you're ready to move on to the next step.
+2. Click <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer-add-cluster">'+'</walkthrough-editor-spotlight> in the title bar of the Clusters explorer to create a new cluster.
 
+3. Choose either **Minikube** or **Google Kubernetes Engine** and follow the instructions below.
 
---- 
 
-### GKE
+### Minikube
```

Comment:
I wonder if it wouldn't be worth adding a tiny disclaimer under this and the next header, e.g. "if you're planning on deploying in GKE, skip this section"

Reply:
That's a good idea. I found it difficult to format the two options in a way that makes it clear the user only chooses one or the other.

Updated with: *(skip this section if you're planning on deploying to GKE)*

Reply:
Should we also add a similar note in the GKE section? (skip this if you're deploying to minikube)

Reply:
Good idea!! Added ✅